x/mongo/driver/connstring/connstring.go

```diff
@@ -553,6 +555,17 @@ func (p *parser) addOption(pair string) error {
 		}
 
 		p.RetryWritesSet = true
+	case "retryreads":
```

Comment:
Can this be unit tested in connstring_test.go?

Reply:
added a test